index.html

```diff
@@ -3,13 +3,14 @@
 <head>
   <meta charset="UTF-8">
   <title>KX Frontend First Task</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.0/css/all.min.css" />
   <link rel="stylesheet" href="styling.css">
 </head>
 
 <body>

```

Comment:
I noticed that the design includes a pink background for the template, but it's currently missing in the implementation. Please make sure to match the background color as shown in the design for visual consistency.